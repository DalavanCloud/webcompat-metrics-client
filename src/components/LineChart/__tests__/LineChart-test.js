/* This Source Code Form is subject to the terms of the Mozilla Public
 * License, v. 2.0. If a copy of the MPL was not distributed with this
 * file, You can obtain one at http://mozilla.org/MPL/2.0/. */

import React from "react";
import renderer from "react-test-renderer";

import LineChart from "..";

jest.mock("react-chartjs-2", () => ({ Line: () => <div>Chart</div> }));

it("renders LineChart default correctly", () => {
  const tree = renderer
    .create(
      <LineChart
        title={"Title LineChart"}
        labels={["1", "2"]}
        data={["1000", "2000"]}
      />,
    )
    .toJSON();
  expect(tree).toMatchSnapshot();
});
