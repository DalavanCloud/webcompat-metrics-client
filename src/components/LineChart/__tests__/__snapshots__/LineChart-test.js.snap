// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`renders LineChart default correctly 1`] = `
<article
  className="container"
>
  <div
    className="wrapper"
  >
    <h3
      className="title"
    >
      Title LineChart
    </h3>
    <div>
      <div>
        Chart
      </div>
    </div>
  </div>
</article>
`;
